orcestra_book/plans/HALO-20240905a.md

```diff
@@ -0,0 +1,230 @@
+---
+arrival_airport: GVAC
+categories: [ec_under]
+crew:
+- job: PI
+  name: Julia Windmiller
+- job: WALES
+  name: Silke Gross
+- job: HAMP
+  name: Yuting Wu
+- job: Dropsondes
+  name: Bjorn Stevens
+- job: Smart/VELOX
+  name: Kevin Wolf
+- job: SpecMACS
+  name: Lea Volkmer
+- job: Flight Documentation
+  name: Martin Wirth
+- job: Ground contact
+  name: TBA
+departure_airport: GVAC
+flight_id: HALO-20240905a
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+landing: 2024-09-05 tba
+orphan: true
+platform: HALO
+takeoff: 2024-09-05 tba
```

Comment:
While I appreciate the honesty in the accuracy of the takeoff and landing times, our scripts need them to be valid ISO 8601 strings 😬.

Reply:
I also already saw that ;-) but was too slow to reply